### PR TITLE
Fix access level for default Kubelet file discovered with CIS 1.8 benchmark 

### PR DESCRIPTION
**Pull request by djsly**
**What type of PR is this?**

/kind bug
**What this PR does / why we need it**:
Changes the access level to 600 for the default kubelet file

From Kube-Bench
[FAIL] 4.1.9 If the kubelet config.yaml configuration file is being used validate permissions set to 600 or more restrictive (Automated)

== Remediations node ==
4.1.9 Run the following command (using the config file location identified in the Audit step)
chmod 600 /etc/default/kubelet

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

**Comment by cameronmeissner** [comment]
I think you're missing a `make generate`

**Comment by Devinwong** [comment]
Why do we need to change the access to 600? Is it to tighten the access level?

**Comment by djsly** [comment]
> Why do we need to change the access to 600? Is it to tighten the access level?
From Kube-Bench
[FAIL] 4.1.9 If the kubelet config.yaml configuration file is being used validate permissions set to 600 or more restrictive (Automated)

== Remediations node ==
4.1.9 Run the following command (using the config file location identified in the Audit step)
chmod 600 /etc/default/kubelet

**Comment by cameronmeissner** [comment]
will let abe2e's run

File: parts/linux/cloud-init/artifacts/cse_config.sh [hunk 1]
```diff
@@ -405,6 +405,7 @@ ensureKubelet() {
     if [ -n "${AZURE_ENVIRONMENT_FILEPATH}" ]; then
         echo "AZURE_ENVIRONMENT_FILEPATH=${AZURE_ENVIRONMENT_FILEPATH}" >> "${KUBELET_DEFAULT_FILE}"
     fi
+    chmod 0600 "${KUBELET_DEFAULT_FILE}"
```
Comment (anujmaheshwari1) [inline parts/linux/cloud-init/artifacts/cse_config.sh:408]: i suppose we do this for all files lower down this function but missed it for /etc/default/kubelet

**Review by anujmaheshwari1 — APPROVED** [review]

**Review by Devinwong — APPROVED** [review]

**Comment by cameronmeissner** [comment]
fixing abe2e's at the moment, then will let it run :)

**Review by cameronmeissner — APPROVED** [review]
all green

**Pull request merged**